src/analysis/GameAnalyzer.cpp

```diff
@@ -44,7 +44,13 @@ std::shared_ptr<AnalysisReport> GameAnalyzer::generateReportNow() {
 }
 
 BallPossession GameAnalyzer::convertPossession(rtt::ai::BallPossession::Possession possession) {
+    /**
+     * Default should usually be the bottom one
+     */
     switch (possession) {
```

Comment:
Discard this change, I'm dumb and made a mistake, my bad

Reply:
Making mistakes does not make you a dumb person :)